pyproject.toml

```diff
@@ -42,8 +42,9 @@ quality = ["black", "ruff", "hf_doc_builder@git+https://github.com/huggingface/d
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-packages = ["QEfficient"]
+[tool.setuptools.packages.find]
```

Comment:
What is the name of the package that get's installed? Is it `QEfficient`?
The name that comes up in `pip3 freeze` ?

Reply:
Yeah, the name QEfficient comes up in the `pip freeze`. The name listed there is not related here. It will use the `project.name` value in the file.